packages/viewer/src/ui/components/lhci-components.jsx

```diff
@@ -9,8 +9,10 @@ export {LoadingSpinner} from '../../../../server/src/ui/components/loading-spinn
 export {Paper} from '../../../../server/src/ui/components/paper.jsx';
 export {LhrViewerButton} from '../../../../server/src/ui/components/lhr-viewer-button.jsx';
 
+/** @type {string} */
 // @ts-ignore - tsc doesn't get parcel :)
```

Comment:
```suggestion
// @ts-expect-error - tsc doesn't get parcel :)
```
while we're doing this, might as well :)